python/doc/examples/probabilistic_modeling/random_vectors/plot_python_randomvector.py

```diff
@@ -5,12 +5,12 @@
 # %%
 # In this example we are going to create a distribution or copula.
 #
-# The way to go is inheriting the PythonRandomVector class and overload its methods:
+# The way to go is inheriting the :class:`~openturns.PythonRandomVector` class and overload its methods:
 #
-# - getRealization
-# - getSample
-# - getMean
-# - getCovariance
+# - `getRealization`
+# - `getSample`
+# - `getMean`
+# - `getCovariance`
```

Comment:
The current version does not generate the corresponding links:
![image](https://github.com/openturns/openturns/assets/31351465/45cedce4-37a2-45c3-bb2a-d515cd3a4da9)


```suggestion
# - `:meth:`~openturns.PythonRandomVector.getRealization`
# - `:meth:`~openturns.PythonRandomVector.getSample`
# - `:meth:`~openturns.PythonRandomVector.getMean`
# - `:meth:`~openturns.PythonRandomVector.getCovariance`
```